Gemfile.lock

```diff
@@ -1,26 +1,26 @@
 PATH
```

Comment:
We should get rid of lock file from the repo

Reply:
Done. Let me know if there is something else to do